### ISSUE TITLE
JDK10: Compile failing with missing method getDeclaringClass()

### ISSUE THREAD

**Issue by yongja79** [eclipse/openj9#840]
The JDK10 compile is failing with the error 

```
ERROR: Build failed for target 'all' in configuration 'linux-x86_64-normal-server-release' (exit code 2)
Stopping sjavac server

=== Output from failing command(s) repeated here ===
* For target jdk_modules_java.base__the.java.base_batch:
/root/openJDK10-J9-mirror/src/java.base/share/classes/java/lang/invoke/MethodHandleImpl.java:1792: error: cannot find symbol
                return memberName.getDeclaringClass();
                                 ^
  symbol:   method getDeclaringClass()
  location: variable memberName of type MemberName
Note: Some input files use or override a deprecated API.
```
The OpenJ9 implementation of the class ./jcl/src/java.base/share/classes/java/lang/invoke/MemberName.java doesn't have a getDeclaringClass() that the openJDK does.

**Comment by pshipton** [comment]
@DanHeidinga did you mean to add the tag "depends:openjdk10"?

**Comment by DanHeidinga** [comment]
Nope - meant to tag as `jdk10`.  Thanks for catching and correcting that!

**Comment by JasonFengJ9** [comment]
Will look at this along with #824 and #826.

**Comment by JasonFengJ9** [comment]
This issue can be closed by PR #852.
